setup.py

```diff
@@ -51,7 +51,7 @@ def generate_build_meta(aot_build_meta: dict) -> None:
 
 ext_modules = []
 cmdclass = {}
-install_requires = ["numpy", "torch", "ninja", "requests"]
+install_requires = ["numpy", "torch", "ninja", "requests", "nvidia-nvshmem-cu12", "nvidia-cutlass-dsl"]
```

Comment:
Currently `nvidia-cutlass-dsl` only supports python 3.12 so we might make it default dependency after more wheels for more python versions are ready, otherwise the flashinfer installation might fail (if user are not using python 3.12).